contrib/config/systemd/system/custodia@.service

```diff
@@ -1,14 +1,14 @@
-# /etc/systemd/system/custodia.service
+# /etc/systemd/system/custodia@.service
 
 [Unit]
-Description=Custodia Secrets Service
+Description=Custodia Secrets Service for %I
```

Comment:
https://fedoramagazine.org/systemd-template-unit-files/

tl;dr: %I is quoted name, %i is unquoted name